common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/Model.java

```diff
@@ -54,7 +54,10 @@
         public enum Range {
```

Comment:
I think you can set up final variable for minute, hour and day to make it more readable.